devito/dse/rewriters.py

```diff
@@ -408,5 +418,22 @@ def _extract_sum_of_products(self, cluster, template, **kwargs):
 
 class CustomRewriter(AggressiveRewriter):
 
+    passes_mapper = {
+        'new_pass': AggressiveRewriter._new_pass
+    }
+
+    def __init__(self, passes, template=None, profile=True):
+        try:
+            passes = passes.split(',')
+        except AttributeError:
+            # Already in tuple format
+            if not all(i in CustomRewriter.passes_mapper for i in passes):
+                raise DSEException("Unknown passes `%s`" % str(passes))
+        self.passes = passes
+        self.template = template
+        self.profile = profile
+        # super(CustomRewriter, self).__init__(params)
```

Comment:
why is this commented out?

we should call the superclass constructor and pass in the right values

Reply:
I left it in order not to forget to do the superclass as a reminder. Should have added TODO. I forgot it earlier...

Reply:
👍